### PR TITLE
Adds information about a bootstrapping bug in Juju

### PR DESCRIPTION
**Pull request by Gmerold**
# Description

- Adds information about a bootstrapping bug in Juju 
- Splits Microk8s addons enabling (chaining is going to be deprecated)
- Replaces `relate` with `integrate`

# Checklist:

- [ ] My code follows the [style guidelines](/CONTRIBUTING.md) of this project
- [ ] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that validate the behaviour of the software
- [ ] I validated that new and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [ ] I have bumped the version of the library

### CODE REVIEW

File: docs/tutorial/04_integrate_sdcore_with_observability.md [hunk 1]
```diff
@@ -42,7 +42,7 @@ juju deploy cos-configuration-k8s \
 Relate it to Grafana:
 
 ```console
-juju relate cos-configuration-k8s grafana
+juju integrate cos-configuration-k8s grafana
```
Comment (gruyaume) [inline docs/tutorial/04_integrate_sdcore_with_observability.md:45]: This is already addressed in PR #42

File: docs/tutorial/01_bootstrap_juju.md [hunk 2]
```diff
@@ -27,6 +29,13 @@ sudo snap install juju --channel=3.1/stable
 
 Bootstrap a Juju controller
 
+```{note}
+If you have never had Juju installed on your machine before, prior to bootstrapping 
+a Juju controller, from your terminal, create directories required by Juju:
+mkdir -p /home/ubuntu/.local/share
+This is a workaround to [a Juju bug](https://bugs.launchpad.net/juju/+bug/1988355).
```
Comment (gruyaume) [inline docs/tutorial/01_bootstrap_juju.md:36]:
```suggestion
There is a [bug](https://bugs.launchpad.net/juju/+bug/1988355) in Juju that occurs when bootstrapping a controller on a new machine. If you encounter it, create the following directory:`mkdir -p /home/ubuntu/.local/share`.
```

And this comment should occur "after" the bootstrap in the sense that the user should do this only if the error has occured.

File: docs/.wordlist.txt [hunk 3]
```diff
@@ -11,10 +11,12 @@ Juju
 Kubernetes
 lifecycle
 MicroK
+mkdir
 Multus
 Observability
 ONOS
 ONF's
+ubuntu
```
Comment (gruyaume) [inline docs/.wordlist.txt:19]: You probably won't need these 2 new words if you have the terminal commands inside `single quotes`

**Review by gruyaume — CHANGES_REQUESTED** [review]

File: docs/tutorial/01_bootstrap_juju.md (hunk 2 above)
Comment (Gmerold) [reply docs/tutorial/01_bootstrap_juju.md]: :+1:

File: docs/.wordlist.txt (hunk 3 above)
Comment (Gmerold) [reply docs/.wordlist.txt]: Indeed :) Thanks!

**Review by gruyaume — APPROVED** [review]

**Pull request merged**